### ISSUE TITLE
[🐞] Firefox: 404 partytown-sandbox-sw.html

### ISSUE THREAD

**Issue by sotasan** [QwikDev/partytown#581]
### Describe the bug

Hey,

I have tested around with Partytown a bit, and I have found that if you hard reload in Firefox it can't find the `partytown-sandbox-sw.html`. I have seen there were other issues with the same bug, but they did not have an example, so I have created an Astro project with this bug.

- #441 
- #107 

> There are also quite a lot of warnings in Firefox, but I don't know if this is the intended behaviour for backwards compatibility or something like that. ~Those warnings don't occur in the incognito mode. I also have disabled all my extensions.~ (#582)
> <img width="791" alt="grafik" src="https://github.com/BuilderIO/partytown/assets/39193720/50387176-2b64-4e15-b551-e8664d7c1fc0">

### Reproduction

https://github.com/sotasan/apparent-ablation

### Steps to reproduce

1. `pnpm install`
2. `pnpm build`
3. `pnpm preview`
4. Open http://localhost:4321 in Firefox
5. Open the developer console
6. Go to the Network tab
7. Press `Ctrl + Shift + R`

### Browser Info

Firefox

**Comment by replete** [comment]
I'm also testing partytown and have this same experience.

Also, Firefox doesn't even show the serviceworker when it does work..
![Screenshot 2024-04-13 at 17 06 19](https://github.com/BuilderIO/partytown/assets/812139/5e39e2b4-ab40-4f3f-9e63-157200cd2dda)

**Comment by aaa123eee** [comment]
We're having the very same issue

@sotasan I am curious if you have found the root of the problem since then 🤔

**Comment by sotasan** [comment]
> @sotasan I am curious if you have found the root of the problem since then 🤔

No, unfortunately not. There were also several other issues that I encountered, I have also created issues regarding those. Because of that, I had to remove Partytown from my production projects.
